pynsca.py

```diff
@@ -91,6 +91,21 @@ def _encrypt_packet(self, toserver_pkt, iv, mode, password):
             myDes = Crypto.Cipher.DES3.new(password, Crypto.Cipher.DES3.MODE_CFB,iv)
             toserver_pkt = ''.join(myDes.encrypt(toserver_pkt))
             #print "toserver_pkt: "+toserver_pkt
+        elif mode == 8:
+            import Crypto.Cipher.Blowfish
+            import Crypto.Util.randpool
+
+            max_key_size = Crypto.Cipher.Blowfish.key_size[-1]
+            password = (password + ('\0' * max_key_size))[:max_key_size]
+            iv_size = 8
+            if len(iv) >= Crypto.Cipher.Blowfish.block_size:
+                iv = iv[:iv_size]
```

Comment:
Should what be block size?  Maybe just take the refactoring, that is probably more clear.  This was a copy of the previous DES3 code, so it probably should stay as it is so that it is using the same wording in both places.  But it has been unified in the refactor.